onmt/opts.py

```diff
@@ -495,6 +495,19 @@ def _add_train_general_opts(parser):
     group.add('--master_port', '-master_port', default=10000, type=int,
               help="Port of master for torch.distributed training.")
 
+    # LoRa
+    group.add('--lora_layer', '-lora_layer', default=[], nargs='*', type=str,
```

Comment:
I would put `layers` instead of `layer`, to make it a bit more clear that it can be several.